app/safe/internal/bootstrap/bootstrap.go

```diff
@@ -166,7 +173,7 @@ func CreateCryptoKey(id *string, updatedSecret chan<- bool) {
 
 	if secret != state.BlankAgeKeyValue {
 		log.InfoLn(id, "Secret has been set in the cluster, will reuse it")
-		state.SetAgeKey(secret)
+		state.SetMasterKey(secret)
```

Comment:
This is the master encryption key; “Age” is just an implementation detail. No need to leak it out in the signature.